script/run-cypress-tests-gha.sh

```diff
@@ -7,11 +7,10 @@ else
   # Flags to indicate to tests that they are running in a CI environment
   # and also that they are running in CircleCI specifically.
   export CYPRESS_CI=$CI
-  export CYPRESS_CIRCLECI=$CIRCLECI
 
   # Use mocha-junit-reporter and save results in './test-results'.
   reporterArgs="--reporter cypress-multi-reporters --reporter-options \"configFile=config/cypress-reporters.json\""
 
-  # Start the web server & run Cypress tests.
-  yarn start-server-and-test "node src/platform/testing/e2e/test-server.js --buildtype=vagovprod --port=3001 > /dev/null 2>&1" :3001 "yarn cy:run $reporterArgs"
+  # Run Cypress tests.
+  yarn cy:run $reporterArgs
```

Comment:
is this script really needed any more, or can we put the args in `cy:test:gha`?

Reply:
Probably can pass the args directly to the NPM script. This was originally needed for that conditional when there were no Cypress tests at all. We've certainly come a long way!

Reply:
This script will likely be modified when we parallelize Cypress tests on GHA.